build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -238,8 +238,11 @@ service ActionCache {
 //   the uploaded data once uncompressed, and MUST return an
 //   `INVALID_ARGUMENT` error in the case of mismatch.
 //
-// Note that when writing compressed blobs, the `WriteRequest.write_offset`
-// refers to the offset in the uncompressed form of the blob.
+// Note that when writing compressed blobs, the `WriteRequest.write_offset` in
+// the initial request refers to the offset in the uncompressed form of the
+// blob. In subsequent requests, `WriteRequest.write_offset` MUST be the sum of
+// the previous `WriteRequest.write_offset` and the size of the compressed chunk
+// in the previous request.
```

Comment:
oh, I think this should be "the current request" - otherwise lgtm

Reply:
It shouldn't though, e.g. the second chunk should be initial writeOffset (typically 0) + size of the first compressed chunk (in the first request, which is the prev request)

Reply:
D'oh- right.